generator/Makefile

```diff
@@ -118,6 +119,10 @@ $(MIBDIR)/IANA-CHARSET-MIB.txt:
 	@echo ">> Donwloading IANA charset MIB"
 	@curl -s -o $(MIBDIR)/IANA-CHARSET-MIB.txt -L $(IANA_CHARSET_URL)
 
+$(MIBDIR)/IANA-IFTYPE-MIB.txt:
+	@echo ">> Donwloading IANA ifType MIB"
```

Comment:
Why we're not using the cisco version deserves a comment

Reply:
I would say we should document the opposite, why we are using Cisco MIBs that are not written by Cisco. :)

Reply:
True, though this isn't a Cisco mib.